### PR TITLE
receive and fallback support

### PR DESCRIPTION
**Pull request by Janther**
solves #242
`receive` was easy but for `fallback` I rely on the `originalText`

### CODE REVIEW

File: src/nodes/FunctionDefinition.js [hunk 1]
```diff
@@ -6,10 +6,21 @@ const {
...
-  return 'function';
+  if (node.isReceiveEther) return 'receive';
+  // The parser doesn't give us any information about the keyword used for the
+  // fallback.
+  // Luckily, `function` and `fallback` have 8 characters so using the
+  // originalText is the next best option.
+  //
+  // An neat idea would be to rely on the pragma and enforce it but for the
+  // moment this will do.
+  return options.originalText.slice(
```
Comment (mattiaerre) [inline src/nodes/FunctionDefinition.js]: should we enforce here that we should return only `function` and `fallback` and not, for instance, `facepalm` (yes it's 8 letters)?

Comment (Janther) [reply src/nodes/FunctionDefinition.js]: Can do that but will mess with coverage though as there’s no chance for us to trigger the parser to have `facepalm` as a function definition

Comment (Janther) [reply src/nodes/FunctionDefinition.js]: ok I updated it to be a bit more explicit

Comment (mattiaerre) [reply src/nodes/FunctionDefinition.js]: hahaha, I've pushed a commit, what do you think? something in between?

Comment (Janther) [reply src/nodes/FunctionDefinition.js]: yup, this way we'll also know if solidity inserts a new keyword as `undefined` doesn't silently print in `concat`

Comment (mattiaerre) [reply src/nodes/FunctionDefinition.js]: I go ahead and merge then ok?

**Pull request merged**